src/openrct2/ride/Vehicle.cpp

```diff
@@ -5069,8 +5069,8 @@ OpenRCT2::Audio::SoundId Vehicle::ProduceScreamSound(const int32_t totalNumPeeps
  */
 GForces Vehicle::GetGForces() const
 {
-    int32_t gForceVert = ((static_cast<int64_t>(0x280000)) * kUnk9A37E4[pitch]) >> 32;
-    gForceVert = ((static_cast<int64_t>(gForceVert)) * kUnk9A39C4[roll]) >> 32;
+    int32_t gForceVert = ((static_cast<int64_t>(0x280000)) * kUnk9A37E4[EnumValue(pitch)]) >> 32;
+    gForceVert = ((static_cast<int64_t>(gForceVert)) * kUnk9A39C4[EnumValue(roll)]) >> 32;
```

Comment:
`kUnk9A37E4` and `kUnk9A39C4` could do with proper names. 

For example:
 `kUnk9A37E4` -> `kVerticalGForcePitchFactor`
 `kUnk9A39C4` -> `kVerticalGForceRollFactor`

(suggestions for better names are welcome)

Reply:
Addressed in https://github.com/OpenRCT2/OpenRCT2/pull/25099